src/main/java/krona/parser/Parser.java

```diff
@@ -0,0 +1,131 @@
+package krona.parser;
+
+import krona.command.AddCommand;
+import krona.command.Command;
+import krona.command.DeleteCommand;
+import krona.command.ExitCommand;
+import krona.command.FindCommand;
+import krona.command.InvalidCommand;
+import krona.command.ListCommand;
+import krona.command.MarkCommand;
+import krona.command.UnmarkCommand;
+import krona.task.Deadline;
+import krona.task.Event;
+import krona.task.Task;
+import krona.task.ToDo;
+
+/**
+ * Handles the parsing of user input and task data for the Krona chatbot.
+ */
+public class Parser {
+
+    /**
+     * Parses the user's input command and returns the corresponding Command object.
+     *
+     * @param input The user input as a string.
+     * @return The Command object corresponding to the user input.
+     */
+    public static Command parse(String input) {
+        String[] words = input.split(" ", 2);
```

Comment:
this is not coding standard related, but i think this may detract from the project scope if the task is, for example, `todo borrow book` (example given from week 2 ip project requirements).